### PR TITLE
EVG-16501: clean up error message style

### PR DESCRIPTION
**Pull request by Kimchelly**
Jira: https://jira.mongodb.org/browse/EVG-16501

Clean up error message style.

### CODE REVIEW

File: cert.go [hunk 1]
```diff
@@ -173,12 +173,12 @@ func (opts *CertificateOptions) CertRequestInMemory() (*pkix.CertificateSigningR
 
 	ips, err := pkix.ParseAndValidateIPs(strings.Join(opts.IP, ","))
 	if err != nil {
-		return nil, nil, errors.Wrapf(err, "problem parsing and validating IPs: %s", opts.IP)
+		return nil, nil, errors.Wrapf(err, "parsing and validating IPs: %s", opts.IP)
```
Comment (julianedwards) [inline cert.go:176]: `"parsing and validating IP '%s'"`

File: cert.go [hunk 2]
```diff
@@ -173,12 +173,12 @@ func (opts *CertificateOptions) CertRequestInMemory() (*pkix.CertificateSigningR
...
 	}
 
 	uris, err := pkix.ParseAndValidateURIs(strings.Join(opts.URI, ","))
 	if err != nil {
-		return nil, nil, errors.Wrapf(err, "problem parsing and validating URIs: %s", opts.URI)
+		return nil, nil, errors.Wrapf(err, "parsing and validating URIs: %s", opts.URI)
```
Comment (julianedwards) [inline cert.go:181]: `"parsing and validating URI '%s'"`

File: cert.go [hunk 3]
```diff
@@ -405,11 +405,11 @@ func (opts CertificateOptions) getOrCreatePrivateKey() (*pkix.Key, error) {
 	if opts.Key != "" {
 		keyBytes, err := ioutil.ReadFile(opts.Key)
 		if err != nil {
-			return nil, errors.Wrapf(err, "problem reading key from %s", opts.Key)
+			return nil, errors.Wrapf(err, "reading key '%s'", opts.Key)
 		}
 		key, err = pkix.NewKeyFromPrivateKeyPEM(keyBytes)
 		if err != nil {
-			return nil, errors.Wrapf(err, "problem getting key from PEM")
+			return nil, errors.Wrapf(err, "getting key from PEM")
```
Comment (julianedwards) [inline cert.go:412]: `errors.Wrap`

File: mongodb_depot.go [hunk 4]
```diff
@@ -158,15 +158,15 @@ func (m *mongoDepot) CheckWithError(tag *depot.Tag) (bool, error) {
...
 	}
 
 	u := &User{}
 	if err = m.client.Database(m.databaseName).Collection(m.collectionName).FindOne(m.ctx, bson.D{{Key: userIDKey, Value: name}}).Decode(u); err != nil {
 		if err == mongo.ErrNoDocuments {
-			return nil, errors.Wrapf(err, "could not find %s in the database", name)
+			return nil, errors.Wrapf(err, "name '%s' not found", name)
 		}
-		return nil, errors.Wrapf(err, "problem looking up %s in the database", name)
+		return nil, errors.Wrapf(err, "looking up '%s' in the database", name)
```
Comment (julianedwards) [inline mongodb_depot.go:169]: `"looking up name '%s' in the database"`

**Review by julianedwards — CHANGES_REQUESTED** [review]

File: cert.go (hunk 2 above)
Comment (Kimchelly) [reply cert.go]: Fixed.

File: mongodb_depot.go (hunk 4 above)
Comment (Kimchelly) [reply mongodb_depot.go]: Fixed.

**Review by julianedwards — APPROVED** [review]

**Pull request merged**